app/views/records/edit_fields/_subject.html.erb

```diff
@@ -2,7 +2,7 @@
             as: :multi_value,
             input_html: {
               class: 'form-control',
-              data: { 'autocomplete-url' => "/authorities/search/local/subjects",
+              data: { 'autocomplete-url' => "/authorities/search/mesh",
```

Comment:
Ah, I see what's going on. It would be cool if Hyrax would just make this configurable.

I'm actually now in favor of the commit pattern you've used here. I think maybe an addition to the message for the commit that initially the file verbatim would help. Just something acknowledging that this is the exact file, and that we need to host it in the application so we can customize the authority used.